applications/games_menu/modules/preset.c

```diff
@@ -0,0 +1,1210 @@
+		TSU_OptionGroupAdd(self.async_option, 7, "7");
+		TSU_OptionGroupAdd(self.async_option, 8, "8");
+		TSU_OptionGroupAdd(self.async_option, 16, "16");
+		TSU_OptionGroupSetStates(self.async_option, SA_CONTROL + ASYNC_CONTROL_ID, SA_PRESET_MENU, &menu_data.state_app);
+		TSU_FormAddBodyOptionGroup(form_ptr, self.async_option, 2, 2);
+		TSU_DrawableEventSetClick((Drawable *)self.async_option, &AsyncOptionClick);
+
+		if (self.preset->emu_async)
+		{
+			TSU_OptionGroupSelectOptionByKey(self.async_option, self.preset->emu_async);
+		}
+
+		async_label = NULL;
+	}
+
+	{
+		// BY PASS
+		Label *bypass_label = TSU_LabelCreate(form_font, "BY PASS:", self.body_letter_size, false, false);
+		TSU_DrawableSetReadOnly((Drawable*)bypass_label, true);
+		TSU_FormAddBodyLabel(form_ptr, bypass_label, 1, 3);
+
+		self.bypass_option = TSU_CheckBoxCreate(form_font, (uint)self.body_letter_size, 50, self.body_height_size);
+		TSU_DrawableSetId((Drawable *)self.bypass_option, BYPASS_CONTROL_ID);
+
+		TSU_FormAddBodyCheckBox(form_ptr, self.bypass_option, 2, 3);
+		TSU_DrawableEventSetClick((Drawable *)self.bypass_option, &ByPassOptionClick);
+
+		if (self.preset->alt_read)
+		{
+			TSU_CheckBoxSetOn(self.bypass_option);
+		}
+
+		bypass_label = NULL;
+	}
+
+	{
+		// IRQ
+		Label *irq_label = TSU_LabelCreate(form_font, "IRQ:", self.body_letter_size, false, false);
+		TSU_DrawableSetReadOnly((Drawable*)irq_label, true);
+		TSU_FormAddBodyLabel(form_ptr, irq_label, 1, 4);
+
+		self.irq_option = TSU_CheckBoxCreate(form_font, (uint)self.body_letter_size, 50, self.body_height_size);
+		TSU_DrawableSetId((Drawable *)self.irq_option, IRQ_CONTROL_ID);
+
+		TSU_FormAddBodyCheckBox(form_ptr, self.irq_option, 2, 4);
+		TSU_DrawableEventSetClick((Drawable *)self.irq_option, &IRQOptionClick);
+
+		if (self.preset->use_irq)
+		{
+			TSU_CheckBoxSetOn(self.irq_option);
+		}
+
+		irq_label = NULL;
+	}
+
+	{
+		// OS
+		Label *os_label = TSU_LabelCreate(form_font, "OS:", self.body_letter_size, false, false);
+		TSU_DrawableSetReadOnly((Drawable*)os_label, true);
+		TSU_FormAddBodyLabel(form_ptr, os_label, 1, 5);
+
+		self.os_option = TSU_OptionGroupCreate(form_font, (uint)self.body_letter_size, 130, self.body_height_size);
+		TSU_DrawableSetId((Drawable *)self.os_option, OS_CONTROL_ID);
+
+		TSU_OptionGroupAdd(self.os_option, 0, "AUTO");
+		TSU_OptionGroupAdd(self.os_option, 1, "KATANA");
+		TSU_OptionGroupAdd(self.os_option, 2, "HOMEBREW");
+		TSU_OptionGroupAdd(self.os_option, 3, "WinCE");
+		TSU_OptionGroupSetStates(self.os_option, SA_CONTROL + OS_CONTROL_ID, SA_PRESET_MENU, &menu_data.state_app);
+		TSU_FormAddBodyOptionGroup(form_ptr, self.os_option, 2, 5);
+		TSU_DrawableEventSetClick((Drawable *)self.os_option, &OSOptionClick);
+
+		if (self.preset->bin_type)
+		{
+			TSU_OptionGroupSelectOptionByKey(self.os_option, self.preset->bin_type);
+		}
+
+		os_label = NULL;
+	}
+
+	{
+		// LOADER
+		Label *loader_label = TSU_LabelCreate(form_font, "LOADER DEVICE:", 15, false, false);
```

Comment:
Better name - loader fw / loader firmware.
It's not only about device now.
Also currently I've added to ISO Loader firmware file selector for this.

Reply:
Ok perfect, LOADER FW.